lib/elixir/lib/kernel/parallel_compiler.ex

```diff
@@ -226,6 +226,12 @@ defmodule Kernel.ParallelCompiler do
         end
         spawn_compilers(state)
 
+      {:warning, file, line, message} ->
+        if callback = Keyword.get(options, :each_warning) do
```

Comment:
Please make sure to document the `:each_warning` option in the `files/2` documentation.